Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -792,11 +782,11 @@ public virtual void OnClientSceneChanged(NetworkConnection conn)
         // their functionality, users would need override all the versions. Instead these callbacks are invoked
         // from all versions, so users only need to implement this one case.
 
-        public virtual void OnStartHost() {}
-        public virtual void OnStartServer() {}
-        public virtual void OnStartClient(NetworkClient client) {}
-        public virtual void OnStopServer() {}
-        public virtual void OnStopClient() {}
-        public virtual void OnStopHost() {}
+        public virtual void OnStartHost() { }
```

Comment:
why is this better?
is this some kind of convention, or your preference?

Reply:
Convention I think - Visual Studio automatically puts the space in there when you type them, when you Ctrl-K+D, etc.

Reply:
could we please keep {} instead of { } ?
looking at this makes me really anxious for some reason. it's feels like using OnStopServer( ) instead of OnStopServer().

Reply:
Theres a { } in NetworkSyncList you may want to snag when you get rid of them.

Reply:
If you can enforce that in editorconfig or github, then fine, I don't care, but editors will keep putting the spaces in there as their normal settings.